api/src/identity-access-management/domain/usecases/self-delete-user-account.usecase.js

```diff
@@ -10,15 +11,33 @@ import { ForbiddenAccess } from '../../../shared/domain/errors.js';
  * @param{PrivacyUsersApiRepository} privacyUsersApiRepository
  * @returns {Promise<boolean>}
  */
-export const selfDeleteUserAccount = async function ({ userId, privacyUsersApiRepository }) {
+export const selfDeleteUserAccount = async function ({
+  userId,
+  localeFromHeader,
```

Comment:
Le use case n'est pas censé connaître la provenance de la locale, le `header` ne fait pas parti de la couche domaine, mais de la couche application.
```suggestion
  locale,
```

Reply:
Je viens de modifier le dernier commit pour appliquer un nommage correct comme indiqué.